$OpenBSD$

fix bug 10926 (https://bugs.dolphin-emu.org/issues/10926)
which tries to mmap 16GiB of memory and overshoots what's available?
(even though I do have 16GiB of physical memory)

Index: Source/Core/Common/MemArena.cpp
--- Source/Core/Common/MemArena.cpp.orig
+++ Source/Core/Common/MemArena.cpp
@@ -123,7 +123,7 @@ u8* MemArena::FindMemoryBase()
 #if _ARCH_32
   const size_t memory_size = 0x31000000;
 #else
-  const size_t memory_size = 0x400000000;
+  const size_t memory_size = 0x0b0000000;
 #endif
 
 #ifdef _WIN32
